### PR TITLE
IN 866 Fixes

### PR DESCRIPTION
**Pull request by jlpoltrack**
- Small channel list fix
- Spelling in setup_list

### CODE REVIEW

File: mLRS/Common/fhss.cpp [hunk 1]
```diff
@@ -71,7 +71,7 @@ void FhssBase::generate(uint32_t seed)
         // ensure it is not too close to the previous
         // do only if we have plenty of channels at our disposal
         bool is_too_close = false;
-        if ((config_i != FHSS_CONFIG_433_MHZ) && (k > 0)) { // TODO: use smarter method, e.g., cnt < 2/3
+        if (((config_i != FHSS_CONFIG_433_MHZ) || (config_i !=FHSS_CONFIG_866_MHZ_IN)) && (k > 0)) { // TODO: use smarter method, e.g., cnt < 2/3
```
Comment (olliw42) [inline mLRS/Common/fhss.cpp:74]: shouldn't this be && ??

Comment (jlpoltrack) [reply mLRS/Common/fhss.cpp:74]: I thought there would only be 1 frequency value stored in config_i so we'd have to check for 433 _OR_ 866  IN? I could be wrong :)

**Comment by olliw42** [comment]
yes, config_i can be only one of these values

therefore "not 433_MHZ or not 866_MHZ_IN" will be always true and the condition is obsolete

what we want is "not (433_MHZ or 866_MHZ_IN)" which is equa to "(not 433_MHZ and not 866_MHZ_IN)"

:)

**Comment by jlpoltrack** [comment]
Now I see - updated :)

File: mLRS/Common/fhss.cpp [hunk 2]
```diff
@@ -71,7 +71,7 @@ void FhssBase::generate(uint32_t seed)
         // ensure it is not too close to the previous
         // do only if we have plenty of channels at our disposal
         bool is_too_close = false;
-        if (((config_i != FHSS_CONFIG_433_MHZ) || (config_i !=FHSS_CONFIG_866_MHZ_IN)) && (k > 0)) { // TODO: use smarter method, e.g., cnt < 2/3
+        if (((config_i != FHSS_CONFIG_433_MHZ) && (config_i !=FHSS_CONFIG_866_MHZ_IN)) && (k > 0)) { // TODO: use smarter method, e.g., cnt < 2/3
```
Comment (olliw42) [inline mLRS/Common/fhss.cpp:74]: the () around the config_i checks can go :)

Comment (jlpoltrack) [reply mLRS/Common/fhss.cpp]: Updated

Comment (olliw42) [reply mLRS/Common/fhss.cpp]: not exactly as I had in mind but equally well, makes it also better readable
good

**Comment by olliw42** [comment]
have you tested it?
I'm inclined to merge :)

**Comment by jlpoltrack** [comment]
I only tested that it compiles.  We can give heads up on Discord for further testing :)

**Comment by olliw42** [comment]
we both think it's correct, right?
maybe I'm just hilarious with asking for a test :)

**Comment by jlpoltrack** [comment]
Hehe - alright.  Flashed to E5 Mini:

- 866 Label
- Can switch to 866 sucessfully

image

**Comment by olliw42** [comment]
MANY THX !!!!
merging

**Pull request merged**